README.md

```diff
@@ -49,8 +49,9 @@ The current release uses `OpenTelemetry AWS Lambda Instrumentation` version
 
 This Splunk distribution comes with the following defaults:
 
-- [B3 context propagation](https://github.com/openzipkin/b3-propagation).
-- [Jaeger-Thrift exporter](https://www.jaegertracing.io).
+- `tracecontext,baggage` context propagation
```

Comment:
This particular string might not mean anything to end users. I think we should say something like "W3C specified Trace Context and Baggage propagation".

Reply:
Good point.